understanding/sharding.md

```diff
@@ -11,20 +11,168 @@ You do not need to worry about sharding until your bot hits around 2,400 guilds.
+manager.on('shardCreate', (shard) => console.log(`Shard ${shard.id} launched`));
+```
+
+## Sharing Information Between Shards
+
+{% hint style="info" %}
+Information is not readily available between shards. In order to get or share information across shards, you will need to make use of either `fetchClientValues()` or `broadcastEval()`.
+{% endhint %}
+
+Remember how we were talking about sharding being a method of "splitting" the bot into multiple instances of itself? Because your sharded bot is now in separate, individual instances, things like your adding your total guilds or getting a specific guild are not as simple as they were before. We must now use either [`fetchClientValues`](https://discord.js.org/#/docs/main/master/class/ShardClientUtil?scrollTo=fetchClientValues) or [`broadcastEval`](https://discord.js.org/#/docs/main/master/class/ShardClientUtil?scrollTo=broadcastEval) to get information from across shards.
+
+These two functions are your go-to for getting any information from other shards, so get familiar with them!
+
+## FetchClientValues
+
+[`fetchClientValues`](https://discord.js.org/#/docs/main/master/class/ShardClientUtil?scrollTo=fetchClientValues) gets Client properties from all shards. This is what you should use when you would like to get any of the nested properties of the Client, such as `guilds.size` or `uptime`. It's useful for getting things like Collection sizes, basic client properties, and unprocessed information about the client.
+
+Example:
+```javascript
+/*
+    Example result of fetchClientValues() on a bot with 4,300 guilds split across 4 shards.
+    Assume this is being executed on shard 0, the first shard.
+*/
+
+// If we just get our client.guilds.cache.size, it will return
+// only the number of guilds on the shard this is being run on.
+console.log('client.guilds.cache.size');
+// 1050
+
+// If we would like to get our client.guilds.cache.size from all
+// of our shards, we must make use of fetchClientValues().
+const res = await client.shard.fetchClientValues('guilds.cache.size');
+console.log(res);
+// 	Array: [
+//		1050,	// shard 0
+//		1100,	// shard 1
+//		1075,	// shard 2
+//		1075	// shard 3
+//	]
+
+````
+
+Let's say you want to do something like get your total server count - In a non-sharded environment, this would be as simple as getting the `client.guilds.cache.size`. However in a sharded environment, `client.guilds.cache.size` will return not the total servers your bot is in. Instead it returns only the total number of servers *on this shard*, like in the first part of the example above.
+
+Here's an example of a function that uses `fetchClientValues()` to first get, then add the total number of guilds from *all shards* (i.e. your bot's total guild count):
+
+```javascript
+/*
+  	Example by ZiNc#2032
+  	The following code fetches total combined shards' server counts.
+  
+  	discord.js version 12.x
+  	client = new discordjs.Client()
+
+  	returns Number
```

Comment:
`Number` would refer to the boxed number object instance, it's better to use `number` instead, as it refers to the primitive value instead.

```suggestion
  	returns number
```